cross/python38/Makefile

```diff
@@ -8,26 +8,57 @@ PKG_DIR = Python-$(PKG_VERS)
 
 BUILD_DEPENDS = native/$(PKG_NAME)
 
-DEPENDS  = cross/zlib cross/openssl cross/sqlite cross/readline cross/ncursesw cross/bzip2 cross/xz
+DEPENDS  = cross/zlib cross/sqlite cross/readline cross/ncursesw cross/bzip2 cross/xz
 # required for uuid module
 DEPENDS += cross/libuuid
-# libffi is no longer bundled with python
-DEPENDS += cross/libffi
 
 HOMEPAGE = https://www.python.org/
 COMMENT  = Python Programming Language
 LICENSE  = PSF
 
 GNU_CONFIGURE = 1
-CONFIGURE_ARGS  = --enable-shared --enable-ipv6 --without-ensurepip --enable-loadable-sqlite-extensions
-CONFIGURE_ARGS += ac_cv_buggy_getaddrinfo=no ac_cv_file__dev_ptmx=no ac_cv_file__dev_ptc=no ac_cv_have_long_long_format=yes
+
+CONFIGURE_ARGS  = --enable-shared
+CONFIGURE_ARGS += --enable-ipv6
+CONFIGURE_ARGS += --without-ensurepip
+CONFIGURE_ARGS += --enable-loadable-sqlite-extensions
+CONFIGURE_ARGS += --with-computed-gotos=yes
+CONFIGURE_ARGS += --enable-optimizations
```

Comment:
with optimizations it runs a serie of tests (some fail) and needs more resources to build.

Can we avoid this option for BUILD_DEPENDS and DEPENDS for other packages?
Only the build of spk/python38 should enable optimizations to spare build resources for other builds.

Reply:
1. I'll read a bit on that and confirm if this affect the lto as well.  Any thoughts on keeping one or the later or both?
2. I suggest we do this outside of this PR, perhaps with a focus of migrating all remaining apps to python38.  What do you think?